octavia_proxy/api/v2/controllers/load_balancer.py

```diff
@@ -204,13 +205,19 @@ def post(self, load_balancer):
         self._validate_vip_request_object(load_balancer, context=context)
         self._validate_flavor(driver, load_balancer, context=context)
         self._validate_availability_zone(context.session, load_balancer)
-
+        result = None
         # Dispatch to the driver
-        result = driver_utils.call_provider(
-            driver.name, driver.loadbalancer_create,
-            context.session,
-            load_balancer)
-
+        try:
```

Comment:
Is it good idea to have here try/except block if we already have exception handling in call_provider() method ?

Reply:
an idea...what If we just pass the exception message from neutron and do not create our own... something like:

in ulits.py ```call_provider()```:
```
except openstack_exceptions.ConflictException as e:
            raise exceptions.IPAddressInUse(message=e.message)
```
in exceptions.py:
```
class IPAddressInUse(OctaviaException):
    message = _('%(message)s')
    code = 409
```

BUT perhaps it would be better to have it like this:

in ulits.py ```call_provider()```:
```
except openstack_exceptions.ConflictException as e:
            raise exceptions.OpenstackConflictException(message=e.message)
```

in exceptions.py:
```
class OpenstackConflictException(OctaviaException):
    message = _('%(message)s')
    code = 409
```

What do you think ?